### ISSUE TITLE
Package broken on Core 2 Quad / Ubuntu

### ISSUE THREAD

**Issue by NerdyProjects** [openresty/openresty#651]
Hey there,

running Ubuntu 20.04.1 LTS (Linux 5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020 x86_64 x86_64 x86_64) using  openresty                                  1.17.8.2-1~focal1

I get `openresty[865391] trap invalid opcode ip:7f0ee8daa6a6 sp:7ffe064fea90 error:0 in libluajit-5.1.so.2.1.0[7f0ee8daa000+60000]` when I try to start openresty.

I installed via the official apt source `deb http://openresty.org/package/ubuntu focal main`

Any help? :-)

CPU: Intel(R) Core(TM)2 Quad CPU    Q9400  @ 2.66GHz

**Comment by NerdyProjects** [comment]
After some more research it seems the packages are built to require SSE4.2 which is available starting with intel nehalem processors, e.g. the Core-I series (1st generation). I did not verify this; I did try using the `libluajit` package from ubuntu repo, but that doesn't provide all necessary symbols.

Maybe there should be a documentation somewhere that the officially build packages require SSE4.2?

**Comment by agentzh** [comment]
@NerdyProjects Yeah, we are already working on making it select SSE4.2 at runtime using cpuid.
